app/models/katello/rpm.rb

```diff
@@ -215,6 +216,19 @@ def self.applicable_to_hosts(hosts)
         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).distinct
     end
 
+    # Return RPMs that are not installed on a host, but could be installed
+    # the word 'installable' has a different meaning here than elsewhere
```

Comment:
'Installable' (and 'applicable') have essentially the same meaning in our code base as they do for errata-- 'applicable' means there is a package installed on the host that needs an upgrade, and 'installable' means it's both applicable and available to upgrade based on the host's CV/LCE.  However, here I needed a third term that means "_not_ installed right now, but _could_ be."  I coined "yum_installable" because the idea is that you could run "yum install <package_name>" on the host and it would be successful.  I named everything here accordingly, but suggestions to improve the name are welcome.